localstack/services/sqs/models.py

```diff
@@ -1083,7 +1086,8 @@ def receive(
             self.inflight_groups.add(group)
 
             if group.empty():
-                # this can be the case if all messages in the group are still invisible
+                # this can be the case if all messages in the group are still invisible or
+                # if all messages of a group have been processed.
```

Comment:
While investigating I saw that the timeout code in this block is not used later on, should we remove it?